k8sclient/k8sclient.go

```diff
@@ -285,7 +285,7 @@ func getKubernetesDelegate(client *ClientInfo, net *types.NetworkSelectionElemen
 		return nil, resourceMap, err
 	}
 
-	delegate, err := types.LoadDelegateNetConf(configBytes, net, deviceID)
+	delegate, err := types.LoadDelegateNetConf(configBytes, net, deviceID, resourceName)
```

Comment:
I must be missing something fundamental, but... Where is this set in advance? Wouldn't this be unassigned?

Reply:
I'm reading this wrong. I looked at the whole file, ignore.

Reply:
Was just getting ready to comment that it is set above in this function:
`resourceName, ok := customResource.GetAnnotations()[resourceNameAnnot]`